### PR TITLE
utils_libvirt/libvirt_disk: add function to get disk by serial

### PR DESCRIPTION
**Pull request by smitterl**
Use the SERIAL to identify a disk and return its path.

Suggested-by: Chunfu Wen <chwen@redhat.com>
Signed-off-by: Sebastian Mitterle <smitterl@redhat.com>

Used in https://github.com/autotest/tp-libvirt/pull/5323

### CODE REVIEW

File: virttest/utils_libvirt/libvirt_disk.py [hunk 1]
```diff
@@ -24,6 +24,37 @@
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def get_new_disk(session):
+    """
+    Identifies the single disk below /dev/
+    that's not mounted currently.
+
+    :param session: VM session
+    :param test: Test instance
```
Comment (chunfuwen) [inline virttest/utils_libvirt/libvirt_disk.py:33]: here there is no test parameter, others LGTM

Comment (smitterl) [reply virttest/utils_libvirt/libvirt_disk.py]: Thanks for spotting it, fixed.

**Comment by smitterl** [comment]
@chunfuwen There was a problem with my implementation. At some point the 'new_disk' might also have been mounted during the tests. So, here I changed the patch to identifying with the serial as proposed by you https://github.com/autotest/tp-libvirt/pull/5323#discussion_r1416578287

Please can you review again?

**Comment by smitterl** [comment]
Using the serial doesn't work for LUNS:
"error: unsupported configuration: scsi-block 'lun' devices do not support the serial property"

**Pull request closed without merging**